core/src/main/java/com/linecorp/armeria/server/DefaultRoutingContext.java

```diff
@@ -178,10 +202,13 @@ static boolean equals(RoutingContext self, @Nullable Object obj) {
 
         final RoutingContext other = (RoutingContext) obj;
         return self.virtualHost().equals(other.virtualHost()) &&
-               self.method().equals(other.method()) &&
+               self.method() == other.method() &&
                self.path().equals(other.path()) &&
-               Objects.equals(self.contentType(), other.contentType()) &&
-               self.acceptTypes().equals(other.acceptTypes());
```

Comment:
Question: I wonder why `contentType` and `acceptTypes` are removed? 😀

Reply:
Do we ignore them from the cache keys?

Reply:
Because I added `headers()` instead. `contentType` and `acceptTypes`  are included in the headers so I think it would be okay to remove them. Please let me know if you have any concern about that. :-)

Reply:
> Because I added headers() instead.

I didn't catch it. 😅

> contentType and acceptTypes are included in the headers

That sounds reasonable to me. 👍 LGTM